src/toolbars/ToolsToolBar.cpp

```diff
@@ -194,7 +196,7 @@ void ToolsToolBar::Populate()
 {
    SetBackgroundColour( theTheme.Colour( clrMedium  ) );
    MakeButtonBackgroundsSmall();
-   Add(mToolSizer = safenew wxGridSizer(2, 3, 1, 1));
+   Add(mToolSizer = safenew wxGridSizer(2, 4, 1, 1));
```

Comment:
This change should also be conditional on EXPERIMENTAL_BRUSH_TOOL

Be sure all such appearance changes in the UI don't happen when the EXPERIMENTAL is off.